libs/application/templates/parental-leave/src/forms/ParentalLeaveForm.ts

```diff
@@ -245,6 +232,106 @@ export const ParentalLeaveForm: Form = buildForm({
             }),
           ],
         }),
+        buildSubSection({
+          id: 'personalAllowanceSubSection',
+          name: m.personalAllowanceName,
+          children: [
+            buildRadioField({
+              id: 'usePersonalAllowance',
+              name: m.usePersonalAllowance,
+              options: [
+                { label: m.yesOptionLabel, value: 'yes' },
```

Comment:
From experience I would recommend extracting string constants like these (`yes`/`no`), since they have to also be referenced from the data schema, avoiding duplication and magic values.